taskcluster/tc-asserts.sh

```diff
@@ -342,6 +342,29 @@ run_netframework_inference_tests()
   assert_working_ldc93s1_lm "${phrase_pbmodel_withlm}" "$?"
 }
 
+run_netcore_inference_tests()
```

Comment:
Please use `${DS_BINARY_FILE}` that you defined earlier

Reply:
`DS_BINARY_FILE` it was used in `ensure_cuda_usage` call, so I assume there is no need to call it twice, so I will just revert that variable

Reply:
> `DS_BINARY_FILE` it was used in `ensure_cuda_usage` call, so I assume there is no need to call it twice, so I will just revert that variable

not sure what you mean here